src/helpers.js

```diff
@@ -228,6 +228,38 @@ function checkSanitizeDeprecation(opt) {
   }
 }
 
+function indentCodeCompensation(raw, text) {
```

Comment:
Can you move this code into the `Tokenizer.fences` function? It doesn't make sense to have it in `helpers` since it is only ever going to be used in one place.